main.py

```diff
@@ -1,10 +1,35 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as config_file:
+        players_data = json.load(config_file)
+
+    for player, data in players_data.items():
+
+        race = Race.objects.get_or_create(name=data["race"]["name"],
+                                          defaults={"name": data["race"]["name"], # noqa
+                                           "description": data["race"]["description"]})[0] # noqa
+        if data["guild"]:
+            guild = Guild.objects.get_or_create(name=data["guild"]["name"],
+                                                defaults={"name": data["guild"]["name"], # noqa
+                                                "description": data["guild"]["description"]})[0] # noqa
+        else:
+            guild = None
+
+        for skill in data["race"]["skills"]:
+            Skill.objects.get_or_create(name=skill["name"],
+                                        defaults={"name": skill["name"],
+                                                  "bonus": skill["bonus"],
+                                                  "race": race})
```

Comment:
The `get_or_create` method should use a field lookup as the first argument, such as `name=skill['name']`. This ensures that the correct `Skill` entry is retrieved or created.